SU2_CFD/include/limiters/CLimiterDetails.hpp

```diff
@@ -74,11 +74,29 @@ namespace LimiterHelpers
     return (y + delta*proj) / (y + 2*proj*proj);
   }
 
+  inline su2double vanAlbadaFunction(su2double proj, su2double delta)
+  {
+    return delta*(proj + delta) / (pow(proj, 2) + pow(delta, 2) + epsilon());
+  }
+
+  inline su2double pipernoFunction(su2double proj, su2double delta)
+  {
+    if (proj*delta <= 0.0) {
+      return 0.0;
+    }
+    else {
+      const su2double r = proj/delta;
+      return (r >= 1.0) ? su2double((3.0*pow(r, 2.0) - 6.0*r + 19.0) / (pow(r, 3.0) - 3.0*r + 18.0))
+                        : su2double(1.0 + (1.5*r + 1.0)*pow(r - 1.0, 3.0));
```

Comment:
Regarding making this branch-less,
r = max(0.0, proj / (delta + eps*sign(delta))); // handles div by zero and the < 0 branch.
return min("0 < r < 1", "1 < r"); // at least as they are currently defined.
where sign(x) = 1 - 2*(x<0);
(you can also leave it to me, for when I update the reconstruction logic in #1022)